### PR TITLE
Services: Some small fixes and cleanups

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: pkg/systemd/services/services.jsx [hunk 1]
```diff
@@ -471,7 +475,7 @@ class ServicesPageBody extends React.Component {
 
                                         return this.getUnitByPath(unit_path);
                                     }, ex => {
-                                        this.props.isLoading(false);
+                                        this.props.setIsLoading(false);
```
Comment (cockpituous) [inline pkg/systemd/services/services.jsx:478]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-060139-e1bb87d4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/services/services.jsx [hunk 2]
```diff
@@ -503,11 +507,11 @@
                                             this.processFailedUnits();
                                         });
                             }, ex => {
-                                this.props.isLoading(false);
+                                this.props.SetIsLoading(false);
```
Comment (cockpituous) [inline pkg/systemd/services/services.jsx:510]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-060139-e1bb87d4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/services/services.jsx [hunk 3]
```diff
@@ -503,11 +507,11 @@
...
                                 this.setState({ error: cockpit.format(_("Listing unit files failed: $0"), ex.toString()) });
                             });
                 }, ex => {
-                    this.props.isLoading(false);
+                    this.props.setIsLoading(false);
```
Comment (cockpituous) [inline pkg/systemd/services/services.jsx:514]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-060139-e1bb87d4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/services/services.jsx [hunk 4]
```diff
@@ -693,7 +686,14 @@ class ServicesPageBody extends React.Component {
                 const timer_unit = systemd_client[this.props.owner].proxy('org.freedesktop.systemd1.Timer', unitNew.path);
                 timer_unit.wait(() => {
                     if (timer_unit.valid)
-                        this.addTimerProperties(timer_unit, path, unitNew);
+                        if (this.addTimerProperties(timer_unit, unitNew)) {
+                            this.setState(prevState => ({
```
Comment (martinpitt) [inline pkg/systemd/services/services.jsx:690]: This isn't useful by itself, but I'll need it for my big commits to rework the whole state management of the Services page, and I'd like to make sure (run CI) to not break anything.

**Review by KKoukiou — APPROVED** [review]

File: pkg/systemd/services/services.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/systemd/services/services.jsx:478]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-080740-f45e3de4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/services/services.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/systemd/services/services.jsx:510]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-080740-f45e3de4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/services/services.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/systemd/services/services.jsx:514]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-080740-f45e3de4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**